c/tests/CMakeLists.txt

```diff
@@ -4,22 +4,28 @@ set(TEST_SOURCES
     c_space.c
     util.c
 )
+
 add_executable(check_atom check_atom.c ${TEST_SOURCES})
-target_link_libraries(check_atom hyperonc-static CONAN_PKG::libcheck)
+include_directories(${HYPERONC_TARGET_DIR})
```

Comment:
1. Should not be needed in general (CMake should automatically include it) at least it works on my machine. Is there really some platform which needs this?
2. If it is needed then one line should be enough it doesn't make sense to copy it in every test entry.